other failing doctests
```
ImportError: cannot import name 'BackslashOperator' from 'sage.misc.misc' (/sage/src/sage/misc/misc.py)
Error: Process completed with exit code 1.
```

--- a/src/sage/misc/all.py
+++ b/src/sage/misc/all.py
@@ -5,12 +5,10 @@ from sage.misc.all__sagemath_objects import *
 from sage.misc.all__sagemath_environment import *
 from sage.misc.all__sagemath_repl import *
 
-from sage.misc.misc import (BackslashOperator,
-                  exists, forall, is_iterator,
-                  random_sublist,
-                  pad_zeros,
-                  newton_method_sizes, compose,
-                  nest)
+from sage.misc.misc import (
+    exists, forall, is_iterator, random_sublist, pad_zeros,
+    newton_method_sizes, compose, nest
+)
 
 from sage.misc.banner import version
 

--- a/src/sage/repl/preparse.py
+++ b/src/sage/repl/preparse.py
@@ -1737,11 +1737,11 @@ def preparse(line, reset=True, do_time=False, ignore_prompts=False,
         8
 
         sage: preparse("A \\ B")
-        'A  * BackslashOperator() * B'
+        'A \\ B'
         sage: preparse("A^2 \\ B + C")
-        'A**Integer(2)  * BackslashOperator() * B + C'
+        'A**Integer(2) \\ B + C'
         sage: preparse("a \\ b \\") # There is really only one backslash here, it is just being escaped.
-        'a  * BackslashOperator() * b \\'
+        'a \\ b \\'
 
         sage: preparse("time R.<x> = ZZ[]", do_time=True)
         '__time__ = cputime(); __wall__ = walltime(); R = ZZ[\'x\']; print("Time: CPU {:.2f} s, Wall: {:.2f} s".format(cputime(__time__), walltime(__wall__))); (x,) = R._first_ngens(1)'
@@ -1852,9 +1852,6 @@ def preparse(line, reset=True, do_time=False, ignore_prompts=False,
     # Calculus functions
     # f(x,y) = x^3 - sin(y)
     L = preparse_calculus(L)
-
-    # Backslash
-    L = re.sub(r'''\\\s*([^\t ;#])''', r' * BackslashOperator() * \1', L)
 
     if do_time:
         # Time keyword

--- a/src/sage/tests/books/computational-mathematics-with-sagemath/linsolve_doctest.py
+++ b/src/sage/tests/books/computational-mathematics-with-sagemath/linsolve_doctest.py
@@ -20,7 +20,7 @@ Sage example in ./linsolve.tex, line 346::
   ....:     A = matrix(QQ, [[1/(i+j-1) for j in [1..n]] for i in [1..n]])
   ....:     y = A*x
   ....:     A[n-1,n-1] = (1/(2*n-1))*(1+1/(10^5))   # modifies the matrix
-  ....:     s = A\y
+  ....:     s = A.solve_right(y)
   ....:     return max(abs(float(s[i]-x[i])) for i in range(0,n))
 
 Sage example in ./linsolve.tex, line 396::

